solidity/contracts/StakeDelegatable.sol

```diff
@@ -33,14 +32,6 @@ contract StakeDelegatable {
         address authorizer;
     }
 
-    modifier onlyOperatorAuthorizer(address _operator) {
-        require(
-            operators[_operator].authorizer == msg.sender,
-            "Not operator authorizer"
-        );
-        _;
-    }
-
```

Comment:
I'm 50-50 on moving this part to `Authorizations`. The function suggests that it's related to authorizations (so +1 for placing it in `Authorizations`). But on the other hand, it requires data from `StakeDelegatable`'s `operators` mapping (a call to an abstract function `authorizerOf` that is implemented here in `StakeDelegatable`).

Reply:
It's not only about modifier name - `authorizeOperatorContract` from `Authorizations.sol` uses this modifier.